frontend_tests/node_tests/postcss.js

```diff
@@ -0,0 +1,79 @@
+const postcss = require('postcss');
```

Comment:
Steve and I talked about this file a little bit and we think we should move it to a new `tools/test-postcss` script in the future. I feel it'd be better to merge this here for now, and we can move it afterwards, mainly because we couldn't agree on what's the best way to handle moving this. It doesn't fit with the code of the `tools/test-tools` command (which runs python tests), but fits with the concept/idea of it.